network/src/modules/1_unique_tracking.rs

```diff
@@ -0,0 +1,33 @@
+use ethabi::ethereum_types::Address;
+use substreams_ethereum::pb::eth::v2::{self as eth};
+use substreams::store;
+use substreams::store::{StoreSet, StoreSetIfNotExists};
+
+use crate::block_handler::BlockHandler;
+use crate::pb::aggregate_data::BigInt;
+use crate::utils::i64_to_str;
+
+#[substreams::handlers::store]
+pub fn unique_tracking(block: eth::Block, output: store::StoreSetI64) {
```

Comment:
which substreams module uses this function? Doesn't seem like this is referenced in `substreams.yaml`

Reply:
Yep, just haven't pushed the config files to remote yet :)